index.html

```diff
@@ -17,128 +18,277 @@
             </li>
          </ul>
     </header>
-    <section class="mobile-landing">
-        <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see</span> <span class="wrapper">you!</span></h1>
-        <p>
-            I'm a software developer! I can help you
-            build a product , feature or website Look
-            through some of my work and
-            experience! If you like what you see and
-            have a project you need coded, don't
-            hestiate to contact me.
+      <section class="mobile-landing">
+         <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see</span> <span class="wrapper">you!</span></h1>
+         <p>
+               I'm a software developer! I can help you
+               build a product , feature or website Look
+               through some of my work and
+               experience! If you like what you see and
+               have a project you need coded, don't
+               hestiate to contact me.
+            </p>
+            <div class="lets-connect">LET'S CONNECT</div>
+            <div class="lets-connect-icons">
+               <ul>
+                  <li><a href="https://twitter.com/Abiy62463489"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+                  <li><a href="https://www.linkedin.com/in/abiy-hussen-aman-194587183/"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
+                  <li><a href="https://github.com/abiy006"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
+               </ul>
+            </div>
+      </section>
+      <section class="card-section">
+         <section class="card-1">
+               <div class="card-1-bg"></div>
```

Comment:
- **[OPTIONAL]** sorry this wasn't mentioned in previous reviews, please be aware that the project has image content and not a decorative image so to improve please add a `<img />` tag instead to all projects.

More information on [When to Use HTML <img> Tag and CSS background-image Property](https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html)